app/views/devise/sessions/new.html.erb

```diff
@@ -73,14 +86,25 @@
   <div class="modal-dialog" role="document">
     <div class="modal-content">
       <div class="modal-header">
-        <h3 class="modal-title" id="exampleModalLabel">Demo Site Reminder</h3>
+        <h3 class="modal-title" id="exampleModalLabel"><b>This site is for TEST purposes only!</b></h3>
       </div>
       <div class="modal-body">
-        Please be aware that diaperbase.org is a demo/test server for <a href="https://diaper.app">diaper.app</a>.
-        Multiple people have access to the test accounts so please do not save or upload any sensitive data.
+        You're visiting diaperbase.org, a demo/test site for the full site at <a href="https://diaper.app">diaper.app</a>.<br>
+        It is not safe to upload, enter or save any sensitive data here.<br>
+        <div class="modal-body-warning-text">
+          To safely login or reset your password, please visit <a href="https://diaper.app/users/sign_in">diaper.app</a>
```

Comment:
```suggestion
          If you meant to login to your live account goto <a href="https://diaper.app/users/sign_in">diaper.app</a>
```
I feel like the word 'safely' is a bit confusing here and doesn't imply why it is safe. Since this is a user error, I would say something to direct them like "If you meant...."